script/chart-museum.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Copyright 2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+CM_PORT=${CM_PORT:-8090}
+CM_USER=${CM_USER:-"admin"}
+CM_PWD=${CM_PWD:-"password"}
+CHART_MUSEUM_NS=${CHART_MUSEUM_NS:-"chart-museum"}
+CM_VERSION=${CM_VERSION:-"3.9.0"}
```

Comment:
As per https://github.com/vmware-tanzu/kubeapps/pull/5119#discussion_r929730823, I think we should use the same `CHART_MUSEUM_` prefix, no?

Reply:
Yep, agree. Changed it to `CHARTMUSEUM_` as in the `e2e-test.sh` file.